sdk/core/logger/review/logger.api.md

```diff
@@ -23,8 +23,12 @@ export interface AzureLogger {
     warning: debug.Debugger;
 }
 
-// @public
-export type AzureLogLevel = "verbose" | "info" | "warning" | "error";
+// Warning: (ae-forgotten-export) The symbol "AZURE_LOG_LEVELS" needs to be exported by the entry point logger.d.ts
+// 
+// @public (undocumented)
+export type AzureLogLevel = {
+    [k in keyof typeof AZURE_LOG_LEVELS]: typeof AZURE_LOG_LEVELS[k];
+}[number];
```

Comment:
I think I just forgot to add the updated api extraction in this commit. I had that fun code in a previous version but deleted it because it depends on rather new TypeScript features.